modules/cluster/src/main/java/io/fluo/cluster/runner/ScanOptions.java

```diff
@@ -40,6 +40,10 @@
   @Parameter(names = {"-h", "-help", "--help"}, help = true, description = "Prints help")
   public boolean help;
 
+  @Parameter(names = {"-esc", "--escape-non-ascii"}, help = true,
+      description = "Hex encode non ascii bytes")
+  public boolean hexEncNonAscii = false;
```

Comment:
I thought about doing something like `--no-escape-non-ascii`, but I didn't like it.  Also the double negative makes my head spin.  Does that translate to escape ascii?